lib/client/src/wilson.js

```diff
@@ -393,16 +393,19 @@
               return deferred.promise;
           }],
           dependencies: ['$q', 'IRouteService', function($q, IRouteService) {
-            var deferred = $q.defer();
+            var loadDependencies = $q.when();
```

Comment:
$q.when() will wrap something, either it be a promise or a value, into a $q thenable promise. This basically works like $q.resolve() where the promise is resolved immediately with some value. In this case, nothing. 

This prevents us from having to use deferred objects to do this.